packages/apps/crm/src/screens/lead/LeadFormScreen.js

```diff
@@ -121,102 +81,122 @@ const LeadFormScreen = ({navigation, route}) => {
               <Picker
                 pickerStyle={styles.civilityPicker}
                 title={I18n.t('Crm_Civility')}
-                onValueChange={setCivility}
+                onValueChange={value =>
+                  handleLeadFieldChange(value, 'titleSelect')
+                }
                 listItems={civilityList}
                 labelField="name"
                 valueField="id"
-                defaultValue={civility}
+                defaultValue={Number(_lead.titleSelect)}
               />
             </View>
             <View style={styles.checkBoxContainer}>
               <StarScore
-                score={score}
+                score={_lead.leadScoringSelect}
                 showMissingStar={true}
-                onPress={setScore}
+                onPress={value =>
+                  handleLeadFieldChange(value, 'leadScoringSelect')
+                }
```

Comment:
I think there is an issue with this component, display is not updated when I change value of leadScoringSelect (but change is correctly taken into account for API request, it is only a display issue)